src/components/button/button.test.js

```diff
@@ -41,5 +41,128 @@ describe('/components/button', () => {
       const url = await page.url()
       expect(url).toBe(baseUrl + href)
     })
+    describe('preventing double clicks', () => {
+      it('does not block button links', async () => {
+        // Links should take a user to another part of the page or url, so debouncing it wouldnt have any impact.
+        // But we can check that we're not blocking any initial clicks.
+
+        await page.goto(baseUrl + '/components/button/link/preview', { waitUntil: 'load' })
+
+        const href = await page.evaluate(() => document.body.getElementsByTagName('a')[0].getAttribute('href'))
+
+        // we need to start the waitForNavigation() before we click on the button
+        // so we return a promise that is fulfilled when the navigation and the click is respectively fulfilled
+        // this is somewhat counter intuitive, as we humans expect to act and then wait for something.
+        await Promise.all([
+          page.waitForNavigation(),
+          await page.click('a[role="button"]')
+        ])
+
+        const url = await page.url()
+        expect(url).toBe(baseUrl + href)
+      })
+      it('does not trigger on buttons with type=button', async () => {
```

Comment:
Same thing applies here – this is just going to be returning because of the lack of `data-prevent-double-click`